spec/rubocop/cop/lint/enable_statement_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+describe RuboCop::Cop::Lint::EnableStatement do
+  subject(:cop) { described_class.new(config) }
+
+  let(:config) { RuboCop::Config.new }
+
+  it 'registers an offense when a cop is disabled and never re-enabled' do
+    expect_offense(<<-RUBY.strip_indent)
+      # rubocop:disable Layout/SpaceAroundOperators
+      x =   0
+      # Some other code
+      ^ Re-enable Layout/SpaceAroundOperators cop with `# rubocop:enable` after disabling it.
```

Comment:
Where will this offense show up in the case of a long code file?

Reply:
At the first character of the last line

Reply:
Would it make more sense to report it at the line of the disable statement?

My original idea was that the issue is that the EOF was reached without the cop being re-enabled

Reply:
My hunch is that marking the `disable` statement might be easier to understand. 🙂